### PR TITLE
Prototype

### PR DESCRIPTION
**Pull request by olbeck**
@surabhitri Can you check to make sure that the shiny app is working on your computer. Here are a few things to check

- Look at data-rodeo/dat-shinyapp.csv and how I made it in making-dat-wrangle/creating-dat-shinyapp.R to ensure I did the inflation adjustment, geography indicator, and total employee imputation correctly.
- CEO Compensation tool is working as designed. Both a suggested pay and the table of similar orgs are being outputted. 
- The Gender pay gap graph is producing graphs that look reasonable. 

If there are any code breaks, mistakes, suggestions you find please let me know. 

The helper functions (form shinyhelper package) currently don't work. I am looking into this. This should not affect the functionality of the tool. It would just be something cool to have if we can get it to work.

### CODE REVIEW

File: making-data-wrangle/creating-dat-shinyapp.R [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  group_by(FIPS) %>%
+  summarise(RUCA = mean(RUCA), .groups = "keep")
+
+
+#merge 
+dat.merge <- merge(dat.with.geo, dat.ruca, by = "FIPS") %>%
+  #make RUCA into rual, suburban, and metropolitin 
+  mutate(LocationType = case_when(RUCA <= 3 ~ "Urban", 
+                                  RUCA > 3 & RUCA < 8 ~ "Suburban", 
+                                  RUCA >= 8 ~ "Rural")) %>%
```
Comment (surabhitri) [inline making-data-wrangle/creating-dat-shinyapp.R:201]: I would put a check/ assert here to make sure the merge was right and no duplicates are present/ rows are deleted while merging

**Review by surabhitri — CHANGES_REQUESTED** [review]
The code is very easy to read through. Really appreciate the naming convention you have followed. I checked the inflation part and that looks fine too. Good work! :)

**Comment by surabhitri** [comment]
- The gender pay gap graph is producing graphs that look reasonable. Although I would change the colors from pink and blue to something else (if not already done)
- I checked the shiny website to see if it returned errors anywhere it shouldn't and it didn't but I will continue to test a few more edge cases
